packages/gatsby-page-utils/src/index.ts

```diff
@@ -1,4 +1,4 @@
 export { validatePath } from "./validate-path"
 export { createPath } from "./create-path"
-export { ignorePath } from "./ignore-path"
+export * from "./ignore-path"
```

Comment:
I wonder whether it wouldn't be better to do explicit exports, rather than star. I guess that's more of an aside in general.

The reasoning is to be able to figure out where each export came from.

Reply:
@pvdz Fixed 👍